### PR TITLE
Minor release for Azure Exporter

### PR DESCRIPTION
**Pull request by lzchen**
The PR as of 07/03/19 merges the current master branch into v0.6.x.
The changelog/versions are changed from 0.x.xdev to the release versions.
Changelog/version in Azure are bumped to the release version (0.3.1)
Django will be done in a different PR.

### CODE REVIEW

File: contrib/opencensus-ext-azure/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 0.2.2
+Released 2019-06-28
```
Comment (c24t) [inline contrib/opencensus-ext-azure/CHANGELOG.md]: Change this to the date the package is released on PyPI.

File: contrib/opencensus-ext-azure/opencensus/ext/azure/common/version.py [hunk 2]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.1'
+__version__ = '0.2.2'
```
Comment (c24t) [inline contrib/opencensus-ext-azure/opencensus/ext/azure/common/version.py]: Micro version updates are usually small and backwards-compatible, this looks like at least a minor-level change to me. https://www.python.org/dev/peps/pep-0440/#semantic-versioning.

File: contrib/opencensus-ext-django/version.py [hunk 3]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.3.1'
+__version__ = '0.3.2'
```
Comment (c24t) [inline contrib/opencensus-ext-django/version.py]: Why put these in the same PR?

**Comment by lzchen** [comment]
Merge after [#713]

**Comment by lzchen** [comment]
The PR as of now merges the current master branch into v0.6.x.
The changelog/versions are changed from 0.x.xdev to the release versions.
Changelog/version in Azure are bumped to the release version (0.3.1)
Django will be done in a different PR.

**Comment by lzchen** [comment]
@reyang Are you able to sign the CLA or does @c24t have to add the label?

**Comment by reyang** [comment]
> @reyang Are you able to sign the CLA or does @c24t have to add the label?

I don't have access to unblock Google CLA. My suggestion is that instead of cherry picking commits, just taking the file changes.

**Comment by songy23** [comment]
The base branch has an unnecessary rule:

> The base branch requires all commits to be signed.

It prevents one user cherry-picking a commit from another user. If we can disable it, this PR should be good to merge.

**Comment by lzchen** [comment]
@songy23 Good to know. I removed the outstanding commit and applied them manually myself. Should be good to merge.

**Pull request closed without merging**